eessi/testsuite/tests/apps/MetalWalls.py

```diff
@@ -45,7 +45,7 @@ class EESSI_MetalWalls_MW(MetalWallsCheck):
 
     scale = parameter(SCALES.keys())
 
-    valid_systems = ['*']
+    valid_systems = [r'-offline']
```

Comment:
Maybe move this to `after_init`, then we cankeep the `EESSI_mixin_validate_item_in_list` as it originally was. Reason is that I'm not sure if the `filter_valid_systems_by_device_type` hook deals well with a `valid_systems` that is _not_ equal to `['*']` - I seem to remember we put the validation in place for this reason...